spec/lib/driver_commission_history_response_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+require 'gman_client'
+require 'vcr'
+require 'httparty'
+require 'webmock'
+require 'support/vcr'
+
+RSpec.describe GmanClient do
+  describe '.driver_commission_histories' do
+    VCR.use_cassette('driver_commission_histories') do
+      response = GmanClient.driver_commission_histories
```

Comment:
Can you remind me why this was outside of a let or subject block? I remember you telling me that it wasn't called, it looks strange to me and would be nice if we could put it inside something. :wink: